frontend/src/modules/editor/selectTheme/views/editorThemeItemView.js

```diff
@@ -24,6 +24,17 @@ define(function(require){
     },
 
     postRender: function() {
+      var previewUrl = '/api/theme/preview/' + this.model.attributes.name + '/' + this.model.attributes.version;
+      var $previewLoc = this.$el.find('.theme-preview');
+
+      $.ajax(previewUrl, {
+        statusCode: {
+          // if a preview image is found
+          200: function() {
+            $previewLoc.prepend('<img src="' + previewUrl + '"alt="Preview image"' + '/>');
```

Comment:
Can the alt text be localised please?

Could also switch to the jQuery object syntax if you fancied it:
```js
$previewLoc.prepend($('<img/>', { src: previewUrl, alt: '...' }));
```